app.js

```diff
@@ -1,11 +1,136 @@
 /* Imports */
 // this will check if we have a user and set signout link if it exists
 import './auth/user.js';
+import { createItem, getListItems, gotIt, renderItem, removeItems, deleteCompleted } from './fetch-utils.js';
 
 /* Get DOM Elements */
 
+const form = document.querySelector('.new-form');
+const deleteItem = document.getElementById('delete');
+const listEl = document.querySelector('.list');
+const errorDisplay = document.getElementById('error-display');
+const deleteComplete = document.getElementById('delete-complete');
+
 /* State */
 
+let items = [];
+let error = null;
+
 /* Events */
+window.addEventListener('load', async () => {
+    const response = await getListItems();
+    error = response.error;
+    items = response.data;
+
+    if (error) {
+        displayError();
+    }
+
+    if (items) {
+        displayItems();
+    }
+});
+
+async function populateItems() {
+    const response = await getListItems();
+    error = response.error;
+    items = response.data;
+
+    if (error) {
+        displayError();
+    }
+
+    if (items) {
+        displayItems();
+    }
+}
```

Comment:
this function is identical to what you're doing on window load. you could just call ```populateItems()``` in the window event listener